Purchases/Networking/Operations/Handling/CustomerInfoResponseHandler.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  CustomerInfoResponseHandler.swift
+//
+//  Created by Joshua Liebowitz on 11/18/21.
+
+import Foundation
+
+class CustomerInfoResponseHandler {
```

Comment:
I created a few classes like this where responses were handled the same between a couple operations. That way we can inject the handling.